### DISCUSSION TITLE
Home assistant con dominio e configurazione Cloudflare integrazione Google Assistant

### DISCUSSION THREAD

**Discussion by syntesys** [indomus/forum#2662] [HUB: Home Assistant]
Ciao a tutti, 

c'è qualcuno che usa Home Assistant con dominio e configurazione Cloudflare e integrazione con Google Assistant?

Ho già seguito le guide varie più volte (sia quelle di indomus che quelle di home assistant) ma, ogni volta che provo a lanciare un sync, ricevo in ritorno un bel 404, senza nulla di più dettagliato:

`2023-10-17 22:57:50.154 ERROR (MainThread) [homeassistant.components.google_assistant.http] Request for https://homegraph.googleapis.com/v1/devices:requestSync failed: 404`

Ho switchato oggi alla configurazione Cloudflare, considerato che DuckDNS mi stava dando problemi. 

Avete suggerimenti in merito?

Grazie!

**Comment by indomus** [comment]
sicuro di non aver indicato l'indirizzo sbagliato?

**Reply by snakuzzo** [reply]
@syntesys87 dai un'occhiata [QUI](https://community.home-assistant.io/t/google-assistant-https-homegraph-googleapis-com-v1-devices-requestsync-failed-404/504048) e soprattutto [QUI](https://developers.home.google.com/cloud-to-cloud/integration/request-sync#homegraph-api)

**Reply by syntesys** [reply]
Ho fatto un paio di prove stamattina, da home assistant ora viene sincronizzato ma ricevo un errore quando vado ad associare il mio progetto su Google Home. All'inizio, ricevo un errore "Could not reach [test] miaapp", poi riprovando ricevo l'errore "No compatible devices found". La questione è che questa istanza di home assistant ha funzionato finora con duckdns, ma ho switchato a cloudflare (sia dominio che proxy) perché avevo un sacco di down con duckdns.
Dai log di home assistant e da Homegraph API vedo che le richieste arrivano, ma non riesco a capire come e dove guardare per la connessione da google home (giusto per capire dove si sta piantando).

Cosa ne pensate?
Grazie!
